SwiftCurrentLint/.swiftlint.yml

```diff
@@ -132,7 +132,7 @@ excluded: # paths to ignore during linting. Takes precedence over `included`.
   - Workflow/Package.swift
 
 line_length:
-  warning: 191
+  warning: 196
```

Comment:
@wiemerm You good on this?  I'm ok with it.

Reply:
Yes, I'm good with this.